s3_storage_client.go

```diff
@@ -38,11 +39,34 @@ type s3Location struct {
 	s3Path     string
 }
 
+var S3LoggingMode aws.ClientLogMode
+
 func (util *snowflakeS3Client) createClient(info *execResponseStageInfo, useAccelerateEndpoint bool) (cloudClient, error) {
 	stageCredentials := info.Creds
-	var resolver s3.EndpointResolver
+	s3Logger := logging.LoggerFunc(func(classification logging.Classification, format string, v ...interface{}) {
```

Comment:
`v` arg is supposed to resemble `values`, correct? What is it exactly?

Reply:
Fixed after discussion.